btcwallet_test.go

```
@@ -41,50 +40,99 @@ func TestWallet(t *testing.T) {
 		},
 	}
 
-	for _, tt := range tests {
-
-		t.Run(tt.name, func(t *testing.T) {
-
-			err := w.Deposit(tt.dep)
-			if err == ErrNegativeDep {
-				fmt.Println(err)
-				return
-			}
-
-			err = w.Withdraw(tt.with)
-			if err == ErrNegativeWith {
-				fmt.Println(err)
-				return
-			}
-			if err == ErrInsufFunds {
-				fmt.Println(err)
-				return
-			}
-
-			result := w.Balance()
-			if result != tt.exp {
-				t.Errorf("Expected %g, got %g", tt.exp, result)
-			}
-		})
+	t.Run(tests[0].name, func(t *testing.T) {
+
+		w := NewWallet()
+
+		_ = w.Deposit(tests[0].dep)
+		_ = w.Withdraw(tests[0].with)
```

Almost never a good idea to ignore errors :)